FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +20,40 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let this = self else { return }
```

Comment:
Instead of using `this` it would be clearer to use `self` for this attribution.
```swift
	guard let self = self else { return }
```
But following from my previous comment there's no reason to keep a reference to `self`, so we can just check for `nil`:
```suggestion
			guard self != nil else { return }
```